docs/guides/middleware.md

```diff
@@ -37,6 +38,8 @@ videojs.use('*', function(player) {
 });
 ```
 
+Alternative, in versions greater than 7.0.5, you may omit `setSource` entirely and the middleware will get selected implicitly.
```

Comment:
I would just put this above the explanation of how to pass along the source, since it'll be default behavior from now on